demo/openssl-qat-engine/Dockerfile

```diff
@@ -1,12 +1,12 @@
 FROM debian:sid as builder
 
-ENV QAT_DRIVER_RELEASE="qat1.7.l.4.3.0-00033"
-ENV QAT_ENGINE_VERSION="v0.5.41"
+ARG QAT_DRIVER_RELEASE="qat1.7.l.4.6.0-00025"
+ARG QAT_ENGINE_VERSION="v0.5.42"
 
 RUN apt-get update && \
     apt-get install -y git build-essential wget libssl-dev openssl libudev-dev pkg-config autoconf autogen libtool gawk && \
     git clone https://github.com/intel/QAT_Engine && \
-    wget https://01.org/sites/default/files/downloads/intelr-quickassist-technology/$QAT_DRIVER_RELEASE.tar.gz && \
```

Comment:
It's bit bad that the URL base part has changed. Maybe the variable needs to be the whole URL instead of just the file name.

Reply:
It's unfortunate that the URL breaks. I found that the older releases cannot be found at all. Only 4.3.0 is there (with a different URL) and 4.4.0 onwards it's like this. Because of that, I prefer to keep it like this. If we get different URLs in the future, we should change.

Reply:
Ok, this is fine.